src/api/app/validators/workflow_filters_validator.rb

```diff
@@ -1,7 +1,7 @@
 class WorkflowFiltersValidator < ActiveModel::Validator
   def validate(record)
     @scm_extractor_payload = record.scm_extractor_payload
-    @workflow_instructions = record.workflow_instructions.with_indifferent_access
+    @workflow_instructions = record.workflow_instructions.deep_symbolize_keys
```

Comment:
`record` refers to an instance of `Workflow` and we already do `deep_symbolize_keys` on `workflow_instructions` in `Workflow.initalize`, so `deep_symbolize_keys` isn't needed here.

```suggestion
    @workflow_instructions = record.workflow_instructions
```